src/components/playbackSettings/playbackSettings.template.html

```diff
@@ -157,6 +157,17 @@ <h2 class="sectionTitle">
         <div class="selectContainer">
             <select is="emby-select" class="selectSkipBackLength" label="${LabelSkipBackLength}"></select>
         </div>
+        <div class="selectContainer">
+            <select is="emby-select" id="selectPreferredTranscodeAudio" label="${LabelSelectPreferredTranscodeAudio}">
+                <option value="auto">${Auto}</option>
```

Comment:
Can we use empty string as key for `Auto`?

Reply:
as in <option value="auto">${Auto}</option> to become <option value="auto"></option>?

Reply:
No
`<option value="">${Auto}</option>`